Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
+RUN openssl genrsa 2048 > ssl-key.pem
+RUN openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem
+
+COPY . /src
+RUN cd /src; npm install
+
+WORKDIR /src/data
```

Comment:
The `WORKDIR` directive is, to my knowledge, intended to be used only once towards the end of a Dockerfile, and is there to specify the directory that the process that ultimately runs in the container should start in. To that end, you should merge this with the line following, such that it starts

    RUN cd /src/data \
     && echo ...